standalone/src/init.ts

```diff
@@ -314,6 +308,16 @@ export default async function init(cfg: InitConfig) {
 
     await cfg.initCb(store, theme, channelID);
 
+    // Sending this in the initCb doesn't work
```

Comment:
Why? :grinning:

Reply:
I knew you'd ask that. I don't know actually, I didn't look into the why because I was stuck on the other issues. But I can go back and investigate.

Reply:
Ok, looked at it -- my mistake, the comment was wrong. It's a refactor, and this way the recording view gets set up correctly. Pre-PR, the recording view wasn't registering itself as a participant (in the store), so the logic for websockets and displaying reactions was breaking (some of the selectors use the current voice channel state, which was null).
I fixed the comment.

Reply:
And, now I've reverted that change. Because for some reason it breaks the widget.